articles/client-platforms/_includes/_rules-introduction.md

```diff
@@ -0,0 +1,18 @@
+Rules are one of the cool features of Auth0. Reason for it's coolness is it's flexibility which gives you the ability to extend what Auth0 has to offer. 
+They are JavaScript functions which works like middlewares. To see a detailed description, please refer to [the full docs](https://auth0.com/docs/rules)
+
+#### Create a rule
+
+To create a rule just go to [rule new page](https://manage.auth0.com/#/rules/new). You can create it from scratch or use an existing template. Let's use the one called `Add country to the user profile`, which will add the country to the user profile.
+
+![Empty rule](/media/articles/rules/rule-choose-add-country-template.png)
+
+
+This rule simply adds a new `country` attribute to the user profile.
+
+
+![Add country rule](/media/articles/rules/rule-create-add-country-country.png)
+
+Save the rule and that's it. Whenever you login, the rule will be executed, and the country will be added. 
+
+Done! You have added a rule to your Auth0 authentication flow.
```

Comment:
I think we should avoid 'Done!' here since we'll be using it in the end of the tutorial.